setup.py

```diff
@@ -17,7 +17,21 @@ def get_test_suite():
     url='https://github.com/Azure/sonic-utilities',
     maintainer='Joe LeVeque',
     maintainer_email='jolevequ@microsoft.com',
-    packages=['config', 'sfputil', 'show', 'sonic_eeprom', 'sonic_installer', 'sonic_psu', 'sonic_sfp', 'acl_loader', 'sonic-utilities-tests', 'clear', 'debug', 'undebug'],
+    packages=[
+        'acl_loader',
+        'clear',
+        'config',
+        'debug',
+        'pfcwd',
+        'sfputil',
+        'show',
+        'sonic_eeprom',
+        'sonic_installer',
+        'sonic_psu',
+        'sonic_sfp',
+        'sonic-utilities-tests',
+        'undebug',
+        ],
```

Comment:
Please reduce indent to 4 spaces for the closing bracket.

Reply:
Missed this one. Done.